scripts/init/gentoo/gogs

```diff
@@ -1,16 +1,48 @@
 #!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
 
-DIR=/home/git/gogs
 USER=git
-PORT=3000
+DESC="Gogs"
+NAME=gogs
+SERVICEVERBOSE=yes
+PIDFILE=/var/run/$NAME.pid
+SCRIPTNAME=/etc/init.d/$NAME
+WORKINGDIR=/home/git/gogs
+DAEMON=$WORKINGDIR/$NAME
+DAEMON_ARGS="web"
+
+# Exit if the package is not installed
+[ -x "$DAEMON" ] || exit 0
+
+depend() {
+        use logger
+        need net
+}
+
+start() {
+        ebegin "Starting Gogs"
+        sh -c "USER=$USER start-stop-daemon --start --quiet --pidfile $PIDFILE --make-pidfile \\
+                --test --chdir $WORKINGDIR --chuid $USER \\
+                --exec $DAEMON -- $DAEMON_ARGS > /dev/null \\
+                || return 1"
+        sh -c "USER=$USER start-stop-daemon --start --quiet --pidfile $PIDFILE --make-pidfile \\
+                --background --chdir $WORKINGDIR --chuid $USER \\
+                --exec $DAEMON -- $DAEMON_ARGS \\
+                || return 2"
+        eend $?
 
-start_stop_daemon_args="--user ${USER} --chdir ${DIR}"
```

Comment:
These variables (command, command_args, pidfile and so on) are special ones. You shouldn't drop them in favor of writing the same (redundant) thing in start() and stop().